macros/SpecName.ejs

```diff
@@ -1123,17 +1111,17 @@ var specList = {
         name : 'WebRTC 1.0: Real-time Communication Between Browsers',
         url  : 'https://w3c.github.io/webrtc-pc/'
     },
-    'Websockets':{ /* This draft has been merged into HTML WHATWG */
-        name : 'The WebSocket API',
-        url  : 'https://html.spec.whatwg.org/multipage/comms.html'
+    'Websockets':{
+        name : 'HTML (Web sockets)',
+        url  : 'https://html.spec.whatwg.org/multipage/web-sockets.html#network'
     },
     'Web Speech API':{
         name : 'Web Speech API',
         url  : 'https://dvcs.w3.org/hg/speech-api/raw-file/tip/webspeechapi.html'
     },
     'Web Storage':{
-        name : 'Web Storage (Second edition)',
-        url  : 'https://html.spec.whatwg.org/multipage/webstorage.html'
+        name : 'HTML (Web storage)',
+        url  : 'https://html.spec.whatwg.org/multipage/webstorage.html#webstorage'
```

Comment:
:+1: to the name change.
The URL needs to stay without the #webstorage fragment as the're calls like this building a deep link:
{{SpecName('Web Storage', '#dom-storage-length', 'length')}}